documentation/docs/verifiable_credentials/verifiable_presentations.mdx

```diff
@@ -10,17 +10,157 @@ keywords:
+the holder stores it securely until asked to present it.
+A company could then request proof of that university degree: the holder can [create a verifiable presentation](#creation)
+containing their credential, which is already signed by their university, and present it to the company to [validate](#validation).
 
-This example shows how you can create and validate a Verifiable Presentation. A Verifiable Presentation is the format in which you can share a (collection of) Verifiable Credential(s). It is signed by the subject, to prove control over the Verifiable Credential with a nonce or timestamp.
+Note that verifiable presentations that contain personal data should, as with verifiable credentials, be transmitted and stored securely off-chain to satisfy data privacy regulations such as [GDPR](https://gdpr.eu/).
 
+:::note
 
+See the [Verifiable Credentials Data Model Specification](https://www.w3.org/TR/vc-data-model/#presentations) for more information on verifiable presentations.
 
-<CodeSnippet 
+:::
+
+## Creation {#creation}
+
+The IOTA Identity Framework enables holders to construct verifiable presentations easily.
+As demonstrated in the [example](#example), holders need only pass in their credentials to present and sign the presentation.
+
+The following properties may be specified on a presentation:
+
+- [**ID**](https://www.w3.org/TR/vc-data-model/#identifiers): optional URI identifier for the presentation.
+- [**Context**](https://www.w3.org/TR/vc-data-model/#contexts): list of JSON-LD context URIs. Includes `"https://www.w3.org/2018/credentials/v1"` by default.
+- [**Types**](https://www.w3.org/TR/vc-data-model/#types): list of types describing the presentation. Includes `"VerifiablePresentation"` by default.
+- [**Credentials**](https://www.w3.org/TR/vc-data-model/#dfn-verifiable-credentials): list of verifiable credentials to present.
+- [**Holder**](https://www.w3.org/TR/vc-data-model/#dfn-holders): optional URI, typically a DID, of the entity that generated the presentation.
+- [**Refresh Service**](https://www.w3.org/TR/vc-data-model/#refreshing): optional link to a service where the recipient may refresh the included credentials.
+- [**Terms of Use**](https://www.w3.org/TR/vc-data-model/#terms-of-use): optional list of policies defining obligations, prohibitions, or permissions of the presentation recipient.
+
+Of the above, only the list of credentials is required when creating a presentation using the framework.
+However, the holder property should be included to satisfy [subject-holder relationship](#subject-holder-relationship) checks during validation.
+
+After creation, the holder signs the verifiable presentation using a private key linked to one of the verification methods in their
+DID Document and transmits it to a verifier for [validation](#validation).
+
+### Proof Options {#proof-options}
+
+A digital signature on a verifiable presentation both provides data integrity and proves the DID of the holder.
```

Comment:
Proving "control" over a DID specifically has different implications; a signature to assert a claim is different to proving control. I worded it this way because we specifically recommend against relying on such a signature alone for authenticating DIDs.